server/test/repository/ApplicationRepositoryTest.java

```diff
@@ -190,6 +191,16 @@ public void submitApplication_duplicateSubmissionsThrowsException() {
         .isInstanceOf(DuplicateApplicationException.class);
   }
 
+  @Test
+  public void submitApplication_eligibilityDeterminationDefualtNotComputed() {
+    ApplicantModel applicant = saveApplicant("Alice");
+    ProgramModel program = createDraftProgram("Program");
+    ApplicationModel application =
+        repo.submitApplication(applicant, program, Optional.empty()).toCompletableFuture().join();
+    assertThat(
+        application.getEligibilityDetermination().equals(EligibilityDetermination.NOT_COMPUTED));
+  }
```

Comment:
suggestion (maybe blocking): I think you'll want another test case that test that checks that the eligibility is saved correctly when it's not the default. But this connects to my comment in `ApplicationRepository` above, and I need to do some more thinking about how this is all hooked up.

Reply:
If you end up altering the logic in `ApplicationRepository` to set the eligibility determination that is passed in as a param, you can ignore my comment above and just add a test which tests that the param passed in is saved to the application.